src/test/java/com/unit/UNIT/TEST/MachineTest.java

```diff
@@ -47,7 +50,63 @@ public void shouldIncreaseAmountOfMoney2(){
         verify(validator,times(1)).validateMoney(money);
 
     }
-*/
 
+    @Test
+    public void shouldClearAmountOfMoney() {
+        //given
+        int finalResult = 0;
+        //when
+        machine.winInfo();
+        //then
+        assertEquals(machine.bankAmount, finalResult);
+    }
+
+    @Test
+    public void shouldFiveElementsInList() {
+        //given
+        int numberOfElements = 5;
+        //when
+        machine.generateThreeSigns();
+        //then
+        assertEquals(machine.gameSigns.size(), numberOfElements);
+    }
+
+    @Test
+    public void shouldElementsFromZeroToNine() {
+        //given
+        CheckNumberTest checkTest = new CheckNumberTest();
```

Comment:
Stworzyłeś tutaj inny test - nie mamy klasy CheckNumber, dlaczego to powstało? Jeśli chciałbyś zrobić to co mam nadzieję miałeś na myśli to  w klasie testowej (tej tutaj) dodajesz metodę prywatną "check..." (no właśnie tutaj znowu -> słowo check nie mówi innemu programiście  kompletnie co autor miał na myśli, check Money? Check Sign? Check What? )